src/main/groovy/org/transmartproject/batch/concept/ConceptNode.groovy

```diff
@@ -35,6 +35,6 @@ class ConceptNode {
     }
 
     boolean isNew() {
-        code == null
+        i2b2RecordId == null
     }
```

Comment:
Why would this make any difference? The only place `code` is written to is `ConceptTree::reserveIdsFor`, and in that function `i2b2RecordId`. So the behavior should be exactly the same.